autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleClusterV1Controller.java

```diff
@@ -153,7 +153,8 @@ private DistroXAutoscaleClusterResponse updateClusterAutoScaleConfig(Cluster clu
                 asClusterCommonService.createTimeAlerts(cluster.getId(),
                         timeAlertRequestConverter.convertAllFromJson(autoscaleClusterRequest.getTimeAlertRequests()));
                 asClusterCommonService.setAutoscaleState(cluster.getId(), autoscaleClusterRequest.getEnableAutoscaling());
-                asClusterCommonService.setStopStartScalingState(cluster.getId(), autoscaleClusterRequest.getUseStopStartMechanism());
+                asClusterCommonService.setStopStartScalingState(cluster.getId(), autoscaleClusterRequest.getUseStopStartMechanism(),
+                        autoscaleClusterRequest.getTimeAlertRequests() != null && !autoscaleClusterRequest.getTimeAlertRequests().isEmpty());
```

Comment:
Same here: `ObjectUtils#isEmpty` is a nifty method you can use here

Reply:
Done